docs/Dirb.md

```diff
@@ -0,0 +1,36 @@
+# How to install and use Dirb for Mac OS
+
+## Installation 🔧
+
+All you need to do is follow the commands below, and Dirb will be installed in no time!
+
+```sh
+curl -L https://sourceforge.net/projects/dirb/files/dirb/2.22/dirb222.tar.gz/download -o dirb222.tar.gz
+tar xvzf dirb222.tar.gz
+chmod 766 dirb222
+cd dirb222
+find . -type d -exec chmod 766 {} \;
+chmod +x configure
+./configure
+make && make install
+```
+
+---
+
+## How to use Dirb 👨‍💻
+
+For this course, we are providing you with a wordlist of common URL directories, which can be found [here][1], to be used with Dirb. Feel free to use your own though 😉 !
```

Comment:
Is this common.txt similar to the Kali's `/usr/share/dirb/wordlists/common.txt`? It would be nice to mention this and eventually find a link to it. Can we find it on SecLists?

Reply:
It's similar, I got it from `dirbuster` though. I've found `/usr/share/dirb/wordlists/common.txt` in SecLists, I'll substitute it!